content/fr/_TEMPLATE.md

```diff
@@ -9,7 +9,7 @@ category: concept
 
 Un bref descriptif de la technologie ou du concept.
 
-## Problème qu'il adresse
+## Problème auquel il répond
```

Comment:
I think you can rebase and remove this change, since the last PR merged already includes this change